### PR TITLE
Fixed brownie tests

### PR DESCRIPTION
**Pull request by EdNoepel**
**Changes**
- ~Resolved issue pulling collateral from a 100% utilized pool, which would have produced an invalid LUP.~
- ~Unskipped and combined two "purchase" tests into a single test.~ 
- Removed redundant Brownie tests to save maintenance costs.
- Fixed bug converting gas to USD.

### CODE REVIEW

**Comment by grandizzy** [comment]
I think we should be good to remove all brownie tests but test_scaled_pool.py and the rw one (as the others are redundant)

**Comment by EdNoepel** [comment]
> I think we should be good to remove all brownie tests but test_scaled_pool.py and the rw one (as the others are redundant)

Will do.

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Review by grandizzy — APPROVED** [review]

File: tests/conftest.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ class TestUtils:
 
     @staticmethod
     def get_usage(gas) -> str:
-        in_eth = gas * 50 * 10e-9
+        in_eth = gas * 50 * 1e-9
```
Comment (grandizzy) [inline tests/conftest.py:174]: boomer :) so all this time I've been looking at 10x higher prices

**Pull request merged**